content/blog/2021/07/camel-quarkus-release-2.1.0/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Camel Quarkus 2.1.0 Released"
+date: 2021-07-27
+authors: ["zbendiba"]
+categories: ["Releases", "Camel Quarkus"]
+preview: "Camel Quarkus 2.1.0 Released"
+summary: "Camel Quarkus 2.1.0 brings Quarkus 2.1.0.Final"
+---
+
+<sub><sup>Image by <a href="https://www.flickr.com/photos/lanzen/5984113332">Anders Lanzen</a> <a href="https://creativecommons.org/licenses/by-nc-sa/2.0">CC BY-NC-SA 2.0</a></sup></sub>
+
+We are pleased to announce the release 2.1.0 of Camel Quarkus.
+It brings Quarkus 2.1, improved test coverage and one new promoted extension.
+
+Many thanks to all contributors and issue reporters!
+
+## Quarkus 2.1.0.Final
+Please refer to [Quarkus 2.1.0.Final announcement](https://quarkus.io/blog/quarkus-2-1-0-final-released/) for more details.
+
+## New extension
+
+* [OpenTelemetry](/camel-quarkus/latest/reference/extensions/opentelemetry.adoc)
+
+## Serialization support
+We introduced the support for Serialization. Please refer to the section in our [User guide](/camel-quarkus/latest/user-guide/native-mode.html#serialization)
```

Comment:
I wonder whether the serialization support section would be **#registering-classes-for-serialization** instead of **#serialization** ?

Reply:
the Registering classes for reflection has also #reflection  for the section link